logtracer/tracing/tracer.py

```diff
@@ -91,6 +92,15 @@ def start_traced_span(self, incoming_headers, span_name):
             incoming_headers: Incoming request headers. These could be http, or part of a GRPC message.
             span_name (str): Path of the endpoint of the incoming request.
         """
+        if not incoming_headers.get(B3_TRACE_ID):
+            try:
+                traceid, spanid_with_params = incoming_headers.get(B3_GOOGLE_LOAD_BALANCER_TRACE, '').split('/')
+                spanid, params = spanid_with_params.split(';')
+                incoming_headers[B3_TRACE_ID] = traceid
+                incoming_headers[B3_SPAN_ID] = spanid
+            except ValueError:
+                pass
```

Comment:
From what I understand, ` X-Cloud-Trace-Context` is a GCP specific header used in their services. 
I think it might make it clearer what this code is for if we split this out into a separate method.

There's actually a Python implementation with a bit more validation as part of OpenCensus here: https://github.com/census-instrumentation/opencensus-python/blob/1df8f58e55a0dd5eeab991b984420ba7a35721b8/opencensus/trace/propagation/google_cloud_format.py

Reply:
To be honest I think we could probably switch to OpenCensus and move away from logtracer at some point.. I've added in a discussion for coffee morning